src/api/dataops/queries/baseQuery.spec.ts

```diff
@@ -109,17 +113,38 @@ describe("fields method", () => {
 });
 
 describe("compiledRequest method", () => {
-  let { subject } = createSubject();
-  it("should return compiled object", () => {
-    expect((subject as any).compiledRequest).toEqual({
-      action: QueryAction.select,
-    });
+  let subject: any;
+
+  beforeEach(() => {
+    ({ subject } = createSubject());
+  });
+
+  it("should contain correct action key", () => {
+    expect(subject.compiledRequest).toEqual(
+      expect.objectContaining({ action: DEFAULT_ACTION })
```

Comment:
What do you think if we use `Joi` for this? It will make this code much shorter - one `expect` instead of three

Reply:
None of the unit tests uses it (only `e2e`). should I use it?

Reply:
Well,  I do not insist, we can live without it. It's just a pretty weird piece of code for me. What about this one:
```
expect(subject.compiledRequest.action).toEqual(DEFAULT_ACTION);
```
?

Reply:
I like that, done!